zerver/views/registration.py

```diff
@@ -363,28 +378,84 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
+            'password_required': password_auth_enabled(realm) and password_required,
+            'require_ldap_password': require_ldap_password,
+            'password_auth_enabled': password_auth_enabled(realm),
+            'root_domain_available': is_root_domain_available(),
+            'default_stream_groups': get_default_stream_groups(realm),
+            'accounts': get_accounts_for_email(email),
+            'MAX_REALM_NAME_LENGTH': str(Realm.MAX_REALM_NAME_LENGTH),
+            'MAX_NAME_LENGTH': str(UserProfile.MAX_NAME_LENGTH),
+            'MAX_PASSWORD_LENGTH': str(form.MAX_PASSWORD_LENGTH),
+            'MAX_REALM_SUBDOMAIN_LENGTH': str(Realm.MAX_REALM_SUBDOMAIN_LENGTH)
+        }
     )
 
+def notify_support_new_realm(realm: Realm, organization_type: str,
+                             repository_link: str,
+                             organization_nature: str,
+                             organization_website: str,
+                             organization_description: str) -> None:
+
+    context = common_context_unauthed(realm)
+
+    if settings.STAFF_SUBDOMAIN is not None:
+        try:
+            realm_uri = get_realm(settings.STAFF_SUBDOMAIN).uri + reverse('analytics.views.support')
+            staff_realm_uri = realm_uri + "/?q=" + realm.string_id
+            context.update({"realm_uri": staff_realm_uri})
+        except Realm.DoesNotExist:
+            return HttpResponseNotFound("Realm %s does not exist" % (settings.STAFFSUBDOMAIN,))
+
+    if organization_type == "Open-source":
+        context.update({
+            'organization_type': organization_type,
+            'repository_link': repository_link,
+        })
+        send_email(
+            "zerver/emails/organization_info",
+            to_emails=[FromAddress.SUPPORT],
+            from_name="Zulip Account Security",
+            from_address=FromAddress.tokenized_no_reply_address(),
+            context=context
+        )
+    elif organization_type == "Non-profit" or organization_type == "Education":
+        context.update({
+            'organization_type': organization_type,
+            'organization_nature': organization_nature,
+            'organization_website': organization_website,
+        })
+        send_email(
+            "zerver/emails/organization_info",
+            to_emails=[FromAddress.SUPPORT],
+            from_name="Zulip Account Security",
+            from_address=FromAddress.tokenized_no_reply_address(),
+            context=context
+        )
+    elif organization_type == "Other":
+        context.update({
+            'organization_type': organization_type,
+            'organization_description': organization_description,
+        })
+        send_email(
+            "zerver/emails/organization_info",
+            to_emails=[FromAddress.SUPPORT],
+            from_name="Zulip Account Security",
+            from_address=FromAddress.tokenized_no_reply_address(),
+            context=context
+        )
```

Comment:
The send_email thing is copypasted too many times, we can just construct the context in the `ifs` and do `send_email`  at the end.